source/simulator/checkpoint_restart.cc

```diff
@@ -268,6 +268,9 @@ namespace aspect
   void Simulator<dim>::create_snapshot()
   {
     TimerOutput::Scope timer (computing_timer, "Create snapshot");
+    // add the current wall time to the total wall clock time which is stored in wall_time_from_resume
```

Comment:
Better to not use variable names in these comments, otherwise whoever wants to change the variable will have to update the comment
```suggestion
    // add the current wall time to the total wall clock time and reset timer
```

Reply:
It was already out of date. 😉